org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/alloy4/TableView.java

```diff
@@ -35,7 +35,7 @@ public class TableView {
     final static String  SUPERSCRIPTS = "⁰¹²³⁴⁵⁶⁷⁸⁹";
     final static String  SUBSCRIPTS   = "₀₁₂₃₄₅₆₇₈₉";
     final static String  BOX_SINGLE   = "│┌─┬┐┘┴└├┼┤";
-    final static Pattern TABLE_P      = Pattern.compile("\\s*\\{(([\\d\\w$\\s,>\"-]+))\\}\\s*");
+    final static Pattern TABLE_P      = Pattern.compile("\\s*\\{(([\\d\\w$\\s,>\"-/]+))\\}\\s*");
```

Comment:
This is wrong. When you have `<char> '-' <char>` inside a `[]` block you create a range. Make sure the `-` is the last character in the block.